### DISCUSSION TITLE
Playlist

### DISCUSSION THREAD

**Discussion by qwerty08wewg** [gokadzev/Musify#422] [Q&A] [answered]
I wanted to add my playlist that is on YouTube but it tells me to put the playlist ID where I can find it. Then I tried to search for my playlist but I didn't find it, on YouTube it is the playlist is public. I don't know what to do

**Comment by gokadzev** [comment, marked as answer]
> I wanted to add my playlist that is on YouTube but it tells me to put the playlist ID where I can find it. Then I tried to search for my playlist but I didn't find it, on YouTube it is the playlist is public. I don't know what to do

For example let's take this playlist link: https://www.youtube.com/playlist?list=PLPFzKsPt50V8azAzY0Sg2go23AL9nuAAF

https://www.youtube.com/playlist?list= **PLPFzKsPt50V8azAzY0Sg2go23AL9nuAAF** < this is the playlist ID

so you need to copy ID code after list= from the link



or you can use tools like this: https://webutility.io/youtube-playlist-id-extractor

no sign-up or anything, paste link and get playlist ID
